src/Runtime/localtest/src/Views/Home/Index.cshtml

```diff
@@ -109,3 +131,20 @@
   </div>
   }
 }
+
+<script>
+  document.addEventListener('DOMContentLoaded', function() {
+    var appSelect = document.getElementById('AppPathSelection');
+    var prefillLink = document.getElementById('prefillLink');
+    var prefillOrg = document.getElementById('prefillOrg');
+
+    if (appSelect && prefillLink && prefillOrg) {
+      appSelect.addEventListener('change', function() {
+        var selectedApp = appSelect.options[appSelect.selectedIndex].text;
+        prefillLink.href = '/LocalPlatformStorage/blobs/' + selectedApp;
+        var org = selectedApp.split('/')[0];
+        prefillOrg.textContent = org;
+      });
+    }
+  });
+</script>
```

Comment:
Not sure if there is an easy solution, but it would be nice if we would update AuthLevel for the selected app in addition to the LocalPlatformStorage an orgName